actors/common/src/test/java/org/sunbird/feed/FeedUtilTest.java

```diff
@@ -0,0 +1,132 @@
+
+  private ElasticSearchService esUtil;
+  private CassandraOperation cassandraOperation = null;
+  private static Response response;
+  private static IFeedService feedService;
+
+  @Before
+  public void setUp() throws Exception {
+    PowerMockito.mockStatic(FeedServiceImpl.class);
+    PowerMockito.mockStatic(FeedFactory.class);
+    feedService = mock(FeedServiceImpl.class);
+    when(FeedFactory.getInstance()).thenReturn(feedService);
+    when(FeedServiceImpl.getCassandraInstance()).thenReturn(cassandraOperation);
+    when(FeedServiceImpl.getESInstance()).thenReturn(esUtil);
+    when(feedService.getRecordsByProperties(Mockito.anyMap()))
+        .thenReturn(getFeedList(true))
+        .thenReturn(getFeedList(false));
+
+    PowerMockito.mockStatic(ServiceFactory.class);
+    PowerMockito.mockStatic(EsClientFactory.class);
+    PowerMockito.mockStatic(ElasticSearchHelper.class);
+    esUtil = mock(ElasticSearchService.class);
+    esUtil = mock(ElasticSearchRestHighImpl.class);
+    when(EsClientFactory.getInstance(Mockito.anyString())).thenReturn(esUtil);
+
+    cassandraOperation = mock(CassandraOperationImpl.class);
+    response = new Response();
+    Map<String, Object> responseMap = new HashMap<>();
+    responseMap.put(Constants.RESPONSE, Arrays.asList(getFeedMap()));
+    response.getResult().putAll(responseMap);
+    PowerMockito.when(ServiceFactory.getInstance()).thenReturn(cassandraOperation);
+    PowerMockito.when(
+            cassandraOperation.getRecordsByProperties(Mockito.any(), Mockito.any(), Mockito.any()))
+        .thenReturn(response);
+
+    Response upsertResponse = new Response();
+    Map<String, Object> responseMap2 = new HashMap<>();
+    responseMap2.put(Constants.RESPONSE, Constants.SUCCESS);
+    upsertResponse.getResult().putAll(responseMap2);
+    PowerMockito.when(cassandraOperation.upsertRecord(Mockito.any(), Mockito.any(), Mockito.any()))
+        .thenReturn(upsertResponse);
+  }
+
+  private List<Feed> getFeedList(boolean needId) {
+    Feed feed = new Feed();
+    feed.setUserId("123-456-7890");
+    feed.setCategory("category");
+    if (needId) {
+      feed.setId("123-456-789");
+    }
+    Map<String, Object> map = new HashMap<>();
+    List<String> channelList = new ArrayList<>();
+    channelList.add("SI");
+    map.put(JsonKey.PROSPECT_CHANNELS, channelList);
+    feed.setData(map);
+    List<Feed> feedList = new ArrayList<>();
+    feedList.add(feed);
+    return feedList;
+  }
+
+  private Map<String, Object> getFeedMap() {
+    Map<String, Object> fMap = new HashMap<>();
+    fMap.put(JsonKey.ID, "123-456-7890");
+    fMap.put(JsonKey.USER_ID, "123-456-789");
+    fMap.put(JsonKey.CATEGORY, "category");
+    return fMap;
+  }
+
+  @Test
+  public void saveFeedInsertTest() {
+    List<String> userIds = new ArrayList<>();
+    userIds.add("123-456-7890");
+    try {
+      FeedUtil.saveFeed(getShadowUser(), userIds);
+    } catch (Exception e) {
+      Assert.assertTrue(false);
```

Comment:
are u expecting this test case as failure always
becuase your assert will always be false.
add atleast some more test scenario